internal/core/src/segcore/SegmentInterface.cpp

```diff
@@ -293,4 +293,19 @@ SegmentInternalInterface::timestamp_filter(BitsetType& bitset,
     }
 }
 
+const SkipIndex&
+SegmentInternalInterface::GetSkipIndex() const{
+    return skipIndex_;
+}
+
+void
+SegmentInternalInterface::LoadSkipIndex(milvus::FieldId field_id,
+                                        int64_t chunk_id,
+                                        milvus::DataType data_type,
+                                        const void *chunk_data,
+                                        int64_t count) {
+    skipIndex_.Load(field_id, chunk_id, data_type, chunk_data, count);
+}
+
+
```

Comment:
maybe we need a returned value to indicate whether successful or not for loading skipIndex